dz8.py

```diff
@@ -0,0 +1,38 @@
+"""Find max multiplication of three values in the list."""
+
+nums = [-67, -90, -3, -2, -1, -5, -45, -32, -4]
+
+
+def max_multip_of_three(nums):
+    """
+    Create a function to find the maximum multiplication
+    of the three values in the list.
+    """
+    if len(nums) < 3:
+        return 'Not enough numbers in the list'
+
+    nums.sort()
+    print(nums)
+    
+    max_product_1 = nums[-1] * nums[-2] * nums[-3]
+    max_product_2 = nums[0] * nums[1] * nums[-1]
+
+    if max_product_1 > max_product_2 and max_product_1 > 0:
+        return f"Max value = \'{max_product_1}\'. " \
```

Comment:
If function name == max_multip_of_three  then better return not strings but values. In case of error better to do raise ValueError('Your error message'). 
Main idea -> do not mix return values and real errors